client/document/index.jsx

```diff
@@ -188,6 +199,7 @@ class Document extends React.Component {
 					{ // Load GA only if enabled in the config.
 					appConfig( 'google_analytics_enabled' ) && (
 						<script
+							key="__ga"
```

Comment:
Irrelevant [`key`](https://reactjs.org/docs/lists-and-keys.html#keys)? Can we drop it?